src/e2e/puppeteer/__tests__/snapshot.ts

```diff
@@ -287,6 +285,30 @@ describe('Font Size: 22', () => {
     await sleep(400)
   })
 
-  // run the snapshot tests at font size 22
+  // // run the snapshot tests at font size 22
   testSuite()
 })
+
+describe('Drag and Drop simalation tests', () => {
+  beforeEach(removeHUD)
+
+  it('Check simulateDrag and simulateDrop effect on DOM', async () => {
+    await simulateDragAndDrop(true, true)
+
+    await paste(`
+      - a
+      - b
+      - c
+      - d
+    `)
+
+    await dragAndDropThought('a', 'd')
+
+    await sleep(1500)
+
+    const image = await screenshot()
+    expect(image).toMatchImageSnapshot()
+
+    await simulateDragAndDrop(false, false)
```

Comment:
Please add this to the puppeteer [teardown](https://github.com/cybersemics/em/blob/6275018b29631e5d7939fab42e677b8137b8b00d/src/e2e/puppeteer-environment.ts#L46) function.

Reply:
I added this to the teardown function, but it started giving me `window is undefined` for some reason, I gave the first argument a new page using the `browser` but it is giving this error. I am not sure why is this happening, any idea?

Reply:
Hmmm, I'm not sure. If the page is destroyed in teardown, maybe you can simply omit this.

Reply:
All right sure, I will remove it from here.